src/controllers/userController.js

```diff
@@ -26,7 +38,7 @@ const getSingleUser = async (req, res) => {
 				schema: { $ref: '#/definitions/PasswordlessUser' }
 		 }
 		 #swagger.responses[404] = { description: 'No user with id found.' }
-		 #swagger.responses[403] = { description: 'Requester forbidden to fetch this user.' }
```

Comment:
Why was this removed? Isn't it possible to get this error if `checkPermissions` finds that an inappropriate user is making this request?

Reply:
@akosasante  I forgot that checkPermissions also can throw an error, will add it again sorry